test/integration/event-handler-test.ts

```diff
@@ -75,7 +75,7 @@ test("events", (done) => {
       });
 
       eventHandler.on("push", () => {
-        throw new Error("oops");
+        throw new Error("[@octokit/webhooks] oops");
```

Comment:
you don't need the prefix in the test files, only in the code (`src/`). Could you change these back please, to keep the tests simple?

Reply:
Apologies, reverted.